Formula/jenv.rb

```diff
@@ -15,12 +15,23 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+          echo 'status --is-interactive; and source (jenv init -|psub)' >> #{shell_profile}
+          ln -sf #{libexec}/fish/jenv.fish ~/.config/fish/functions/jenv.fish
+          ln -sf #{libexec}/fish/export.fish ~/.config/fish/functions/export.fish
```

Comment:
```suggestion
          ln -sf #{opt_libexec}/fish/*.fish ~/.config/fish/functions
```
I think this should work. We should use `opt_libexec` to avoid referencing full Cellar paths in the caveats (which you'll sometimes see with just `libexec`).

Re `XDG_CONFIG_HOME`, I don't know how variable substitution works in fish, but with bash/zsh:
```suggestion
          ln -sf #{opt_libexec}/fish/*.fish ${XDG_CONFIG_HOME:-$HOME/.config}/fish/functions
```

Reply:
I don't think fish shell has this type of variable defaulting. I've updated with a workaround.

```
ln -sf /opt/homebrew/opt/jenv/libexec/fish/*.fish ${XDG_CONFIG_HOME:-$HOME/.config}/fish/functions

fish: ${ is not a valid variable in fish.
ln -sf /opt/homebrew/opt/jenv/libexec/fish/*.fish ${XDG_CONFIG_HOME:-$HOME/.config}/fish/functions
```